### DISCUSSION TITLE
Freya PrimeVue does not compile SCSS

### DISCUSSION THREAD

**Discussion by petrbouda** [primefaces/community#1150] [PrimeVue Templates]
Hi,

I bought Freya PrimeVue today and did
 
**npm install** 
**npm run dev**

And it look like scss files are not compiled.

![Screenshot from 2024-02-08 16-58-00](https://github.com/primefaces/community/assets/1913914/c619f7e8-8763-4800-890b-b351a0740c3e)

**npm run build** says
/layout/styles/theme/theme-light/teal/theme.css doesn't exist at build time, it will remain unchanged to be resolved at runtime
transforming (370) node_modules/@fullcalendar/interaction/index.js^C

and generates to **dist**:

`
── index.html
└── layout
    ├── images
    │   ├── avatar-m-1@2x.jpg
    │   └── avatar-m-1.jpg
    └── styles
        ├── preloading
        │   ├── preloading.css
        │   └── preloading.scss
        └── theme
            ├── extensions
            │   └── _fullcalendar.scss
            ├── theme-base
            │   ├── _colors.scss
            │   ├── _common.scss
            │   ├── components
            │   │   ├── button
            │   │   │   ├── _button.scss
            │   │   │   ├── _speeddial.scss
            │   │   │   └── _splitbutton.scss
            │   │   ├── data
            │   │   │   ├── _carousel.scss
            │   │   │   ├── _datatable.scss
            │   │   │   ├── _dataview.scss
            │   │   │   ├── _filter.scss
            │   │   │   ├── _orderlist.scss
            │   │   │   ├── _organizationchart.scss
            │   │   │   ├── _paginator.scss
            │   │   │   ├── _picklist.scss
            │   │   │   ├── _timeline.scss
            │   │   │   ├── _tree.scss
            │   │   │   └── _treetable.scss
            │   │   ├── file
            │   │   │   └── _fileupload.scss
            │   │   ├── input
            │   │   │   ├── _autocomplete.scss
            │   │   │   ├── _calendar.scss
            │   │   │   ├── _cascadeselect.scss
            │   │   │   ├── _checkbox.scss
            │   │   │   ├── _chips.scss
            │   │   │   ├── _colorpicker.scss
            │   │   │   ├── _dropdown.scss
            │   │   │   ├── _editor.scss
            │   │   │   ├── _inputgroup.scss
            │   │   │   ├── _inputnumber.scss
            │   │   │   ├── _inputswitch.scss
            │   │   │   ├── _inputtext.scss
            │   │   │   ├── _listbox.scss
            │   │   │   ├── _multiselect.scss
            │   │   │   ├── _password.scss
            │   │   │   ├── _radiobutton.scss
            │   │   │   ├── _rating.scss
            │   │   │   ├── _selectbutton.scss
            │   │   │   ├── _slider.scss
            │   │   │   ├── _togglebutton.scss
            │   │   │   └── _treeselect.scss
            │   │   ├── menu
            │   │   │   ├── _breadcrumb.scss
            │   │   │   ├── _contextmenu.scss
            │   │   │   ├── _dock.scss
            │   │   │   ├── _megamenu.scss
            │   │   │   ├── _menubar.scss
            │   │   │   ├── _menu.scss
            │   │   │   ├── _panelmenu.scss
            │   │   │   ├── _steps.scss
            │   │   │   ├── _tabmenu.scss
            │   │   │   └── _tieredmenu.scss
            │   │   ├── messages
            │   │   │   ├── _inlinemessage.scss
            │   │   │   ├── _message.scss
            │   │   │   └── _toast.scss
            │   │   ├── misc
            │   │   │   ├── _avatar.scss
            │   │   │   ├── _badge.scss
            │   │   │   ├── _blockui.scss
            │   │   │   ├── _chip.scss
            │   │   │   ├── _inplace.scss
            │   │   │   ├── _progressbar.scss
            │   │   │   ├── _progressspinner.scss
            │   │   │   ├── _scrolltop.scss
            │   │   │   ├── _skeleton.scss
            │   │   │   ├── _tag.scss
            │   │   │   └── _terminal.scss
            │   │   ├── multimedia
            │   │   │   ├── _galleria.scss
            │   │   │   └── _image.scss
            │   │   ├── overlay
            │   │   │   ├── _confirmpopup.scss
            │   │   │   ├── _dialog.scss
            │   │   │   ├── _overlaypanel.scss
            │   │   │   ├── _sidebar.scss
            │   │   │   └── _tooltip.scss
            │   │   └── panel
            │   │       ├── _accordion.scss
            │   │       ├── _card.scss
            │   │       ├── _divider.scss
            │   │       ├── _fieldset.scss
            │   │       ├── _panel.scss
            │   │       ├── _scrollpanel.scss
            │   │       ├── _splitter.scss
            │   │       ├── _tabview.scss
            │   │       └── _toolbar.scss
            │   ├── _components.scss
            │   └── _mixins.scss
            ├── theme-dark
            │   ├── avocado
            │   │   └── theme.scss
            │   ├── blue
            │   │   └── theme.scss
            │   ├── _extensions.scss
            │   ├── green
            │   │   └── theme.scss
            │   ├── indigo
            │   │   └── theme.scss
            │   ├── orange
            │   │   └── theme.scss
            │   ├── purple
            │   │   └── theme.scss
            │   ├── red
            │   │   └── theme.scss
            │   ├── teal
            │   │   └── theme.scss
            │   ├── turquoise
            │   │   └── theme.scss
            │   ├── _variables.scss
            │   └── yellow
            │       └── theme.scss
            └── theme-light
                ├── avocado
                │   └── theme.scss
                ├── blue
                │   └── theme.scss
                ├── _extensions.scss
                ├── green
                │   └── theme.scss
                ├── indigo
                │   └── theme.scss
                ├── orange
                │   └── theme.scss
                ├── purple
                │   └── theme.scss
                ├── red
                │   └── theme.scss
                ├── teal
                │   └── theme.scss
                ├── turquoise
                │   └── theme.scss
                ├── _variables.scss
                └── yellow
                    └── theme.scss

`

**Comment by gabrielcastelan** [comment]
@petrbouda Hi, having similar problems here, how did you manage to resolve this?

**Comment by petrbouda** [comment]
Hi, removed primefaces and used Vue with Bootstrap using Claude Code 🙂
